RepoDirectories.props

```diff
@@ -23,7 +23,7 @@
     <BuildToolsTaskDesktopDir>$(ToolsDir)net46/</BuildToolsTaskDesktopDir>
     <BuildToolsTaskDir>$(BuildToolsTaskCoreDir)</BuildToolsTaskDir>
     <BuildToolsTaskDir Condition="'$(MSBuildRuntimeType)' != 'Core'">$(BuildToolsTaskDesktopDir)</BuildToolsTaskDir>
-    <LocalBuildToolsTaskDir>$(LocalToolRuntimePath)</LocalBuildToolsTaskDir>
+    <LocalBuildToolsTaskDir Condition="'$(MSBuildRuntimeType)' == 'Core'">$(LocalToolRuntimePath)netstandard20/</LocalBuildToolsTaskDir>
```

Comment:
Is the condition necessary?  What if a new `MSBuildRuntimeType` is used?  This would be left empty.

Reply:
I guess I can just default it.